## 78% of end-end tests have passed
```
Total executed tests: 23
Failed tests: 4
Expected to fail tests: 1
Passed tests: 18
```
```
IDs of failed tests: 702731,742016,703391,727230 
```
```
IDs of expected to fail tests: 702859 
```
<h3>Failed tests (4)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/watched-account&PR_ID=22390&APK_NAME=22390.apk&TR_CASE_IDS=702731,742016,703391,727230">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830703">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a pinned message by text: Message 4 </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` </div></blockquote></p>

```
private_chats/test_one_to_one_chat.py:267: in test_1_1_chat_pin_messages
    pinned_message.long_press_without_release()
../views/base_element.py:330: in long_press_without_release
    action.click_and_hold(self.find_element()).perform()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkUfE87iJpHZtwsX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScxYZbRsAnnzSLHgr">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830710">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `image-0` </div><div>Device 1: Click system back button </div></blockquote></p>

```
../views/chat_view.py:751: in send_images_with_description
    [self.get_image_by_index(i).click() for i in indexes]
../views/chat_view.py:751: in <listcomp>
    [self.get_image_by_index(i).click() for i in indexes]
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `image-0` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception

During handling of the above exception, another exception occurred:
private_chats/test_one_to_one_chat.py:431: in test_1_1_chat_send_image_save_and_share
    self.chat_1.send_images_with_description(description=image_description, indexes=[0])
../views/chat_view.py:754: in send_images_with_description
    pytest.fail("Can't send image(s) with index(es) %s" % indexes)
E   Failed: Can't send image(s) with index(es) [0]
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkUfE87iJpHZtwsX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScxYZbRsAnnzSLHgr">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830724">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='send']/*[@content-desc='left-icon-for-action']` </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='network-list']/*[@text='Sepolia']` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:81: in test_wallet_send_erc20_from_drawer
    self.wallet_1.send_asset_from_drawer(address=self.receiver['wallet_address'],
../views/wallet_view.py:291: in send_asset_from_drawer
    self.select_network(network_name)
../views/wallet_view.py:258: in select_network
    xpath="//*[@content-desc='network-list']/*[@text='%s']" % network_name).click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='network-list']/*[@text='Sepolia']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESUhaaAmUsDlvfjI6l">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8OmzbouGXOQIuiWq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830712">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 1: Double tap on: Button </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='send']/*[@content-desc='left-icon-for-action']` </div></blockquote></p>

```
wallet_txs/test_wallet_testnet.py:81: in test_wallet_send_erc20_from_drawer
    self.wallet_1.send_asset_from_drawer(address=self.receiver['wallet_address'],
../views/wallet_view.py:290: in send_asset_from_drawer
    self.send_from_drawer_button.double_click()
../views/base_element.py:114: in double_click
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:114: in <listcomp>
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='send']/*[@content-desc='left-icon-for-action']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESUhaaAmUsDlvfjI6l">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8OmzbouGXOQIuiWq">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830709">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for a message by text: description </div><div>Device 2: Find `ChatElementByText` by `xpath`: `//*[starts-with(@text,'description')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` </div></blockquote></p>

```
communities/test_community.py:527: in test_community_one_image_send_reply_set_reaction
    self.channel_2.set_reaction(message=image_description)
../views/chat_view.py:694: in set_reaction
    element.wait_for_element()
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `2`: `Button` by` accessibility id`: `reaction-thumbs-up` is not found on the screen after wait_for_element 
```

<b>[[Might fail from time to time as reaction are set too slowly]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVeiTxtYdudvgoFU0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESM5mFzFJPY4FlgOMO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (18)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830704">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3kPeDRxDTMF26PQB">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830722">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESfnK6rcZPm3WJCSHb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXA4OamEePMRqDpRi">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830721">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESfnK6rcZPm3WJCSHb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXA4OamEePMRqDpRi">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830705">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESf5G1qJCeNoS3dN5O">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yc94JhL8EKhJCgF">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXcUpeC4o0Z1T02Wl">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830723">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESUhaaAmUsDlvfjI6l">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8OmzbouGXOQIuiWq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830711">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESUhaaAmUsDlvfjI6l">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8OmzbouGXOQIuiWq">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830716">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeTyDLicVAXgI1jTW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830714">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeTyDLicVAXgI1jTW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830717">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeTyDLicVAXgI1jTW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830715">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeTyDLicVAXgI1jTW">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830707">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVeiTxtYdudvgoFU0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESM5mFzFJPY4FlgOMO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830706">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVeiTxtYdudvgoFU0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESM5mFzFJPY4FlgOMO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830708">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVeiTxtYdudvgoFU0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESM5mFzFJPY4FlgOMO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830702">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkUfE87iJpHZtwsX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScxYZbRsAnnzSLHgr">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830713">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9yVRiwpcOW51THh9">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830719">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj3ggHJh384Jp4ReA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830718">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj3ggHJh384Jp4ReA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3830720">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj3ggHJh384Jp4ReA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -580,7 +580,7 @@
 
 (rf/reg-sub
  :wallet/token-by-symbol-from-first-available-account-with-balance
- :<- [:wallet/accounts]
+ :<- [:wallet/operable-accounts]
  :<- [:wallet/current-viewing-account-or-default]
  :<- [:wallet/network-details]
  (fn [[accounts {:keys [tokens]} networks] [_ token-symbol chain-ids]]

